recipes-graphics/imx-gpu-viv/imx-gpu-viv-6.inc

```diff
@@ -213,7 +213,9 @@ do_install () {
         # Framebuffer backend for i.MX 8 and beyond
         install -m 0644 ${S}/gpu-core/usr/lib/pkgconfig/egl.pc         ${D}${libdir}/pkgconfig/egl.pc
         cp -r ${S}/gpu-core/usr/lib/wayland/* ${D}${libdir}
+        set -f
```

Comment:
@MaxKrummenacher, could you explain more about the need for `set -f`? I looked at this but couldn't find a reason. So after that, we can extend the commit log.

Reply:
The FILES:xxx variable contains the path to to be deployed files, but does not prefix the path with ${D}.
`for f in ${FILES:libopencl-imx} ${FILES:libopencl-imx-dev} ${FILES:libclc-imx} ${FILES:libclc-imx-dev}; do` will get all elements in the FILES:xxx variables, expand it according to shell rules and the will set each resulting element into `f` and execute in our case the `rm ${D}$f`.

E.g. `${libdir}/libOpenCL${REALSOLIBS}` results in `/usr/lib/libOpenCL.so.*`. If you have on your build host a file or files which matches that pattern `/usr/lib/libOpenCL.so.*` will be replaced by that or those files. Later `rm ${D}$f` will try to remove files with those concreate filenames which likely not be the ones which were installed in the first place.
As a result do_package will the complain about those which were not removed.

If the file expansion didn't find any file matching in the build host filesystem the FILES:xxx entry is passed to `f` with the wildcard and all works. With `set -f` you tell the shell to not expand the wildcards and all works as if there were no matching files in your build host filesystem.